packages/core/src/render3/util/global_utils.ts

```diff
@@ -79,6 +80,7 @@ const globalUtilsFunctions = {
   'getDirectives': getDirectives,
   'applyChanges': applyChanges,
   'isSignal': isSignal,
+  'getDeferBlocks': getDeferBlocks,
```

Comment:
I'd propose adding the `ɵ` for now (to mark it as private), until we get a chance to test it fully in DevTools:
```suggestion
  'ɵgetDeferBlocks': getDeferBlocks,
```